src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,56 @@
+package duke.task;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Class that represents a list that keep tracks
+ * of the tasks that was added, marked, unmarked, or deleted.
+ * @author Oskar Lew
+ * @version 0.1
+ * @since 0.1
+ */
+public class TaskList {
+    private List<Task> taskList = new LinkedList<>();
```

Comment:
Interesting! Is there a reason why you a LinkedList instead of an ArrayList?

Reply:
I agree very interesting indeed.